src/web/test.rs

```diff
@@ -385,10 +384,8 @@ async fn basic_filtered() {
     // Ensure the tile order from adM is preserved
     let tile1 = &tiles[0];
     assert_eq!(tile1["name"], "Acme");
-    assert_eq!(tile1["position"], 0);
     let tile2 = &tiles[1];
     assert_eq!(tile2["name"], "Los Pollos Hermanos");
-    assert_eq!(tile2["position"], 2);
```

Comment:
Do we have any unit test coverage for `position` not being included in the response content? We could invert this and the above change to check for that.

Reply:
Don't the strict integration tests handle this? Adding a unit test for this feels like trying to prove a negative.